### DISCUSSION TITLE
Documented features vs. Test Content

### DISCUSSION THREAD

**Discussion by kirkilj** [jelovirt/org.lwdita#129] [General]
Hi @jelovirt, I noticed about a year ago at one point that there are some features in your test content that don't seem to be documented, such as using HDITA conrefs in Markdown (`<p data-conkeyref="foo/bar">`). If this is still the case, is it a matter of bringing the documentation up to par with the feature set, or are these experimental capabilities? They're not in the DITA-OT docs section on Markdown support. 

Tagging @infotexture.

**Comment by infotexture** [comment]
It may very well be that newer features have been added since the documentation was last updated, as sometimes good things happen when noone's looking.

> is it a matter of bringing the documentation up to par with the feature set

If so, feel free to [file an issue in the docs repo](https://github.com/dita-ot/docs/issues/new/choose), and we'll have a look.
